ambariclient/models.py

```diff
@@ -348,7 +348,7 @@ def stop(self):
 
     def restart(self):
         """Restarts this component on its host, if already installed and started."""
-        if self.state in ('STARTED', 'STOPPED', 'UNKNOWN'):
+        if self.state in ('STARTED', 'STOPPED', 'UNKNOWN', 'INSTALLED'):
```

Comment:
I take it this works ok now then?  It used to complain when you tried to restart something that hadn't been previously started.